vllm/model_executor/weight_utils.py

```diff
@@ -20,6 +20,9 @@
 
 logger = init_logger(__name__)
 
+_xdg_cache_home = os.getenv('XDG_CACHE_HOME', os.path.expanduser('~/.cache'))
```

Comment:
What does `XDG` mean here?

Reply:
I think it just stands for "X Desktop Group" which is the group that set the standard for this https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

To be honest this is a new standard to me, but I took the precedent from the Usage Stats PR https://github.com/vllm-project/vllm/pull/2852 since they use the `XDG_CONFIG_HOME` to control `~/.config`